tests/wicked/before_test.pm

```diff
@@ -54,6 +54,21 @@ sub run {
         systemctl 'start dhcpd.service';
     } else {
         # Common SUT Configuration
+        if (get_var('WICKED_SOURCES')) {
+            zypper_call('--quiet in -t pattern Basis-Devel');
+            zypper_call('--quiet in automake autoconf libtool libnl-devel libnl3-devel libiw-devel dbus-1-devel pkg-config libgcrypt-devel systemd-devel git');
```

Comment:
If is pattern installed why we need to install those specific packages? I would expect that only pattern installation or packages installation should be here. Or pattern is not enough and even all those packages don't pull everything as dependency?

Reply:
second line taken from https://github.com/openSUSE/wicked/blob/master/INSTALL . first line I added because after running test only with second one I got error that gcc is missing . For sure there could be smaller amount of packages installed , maybe just adding gcc to second line would solve the "problem " , but I actually don't what "problem" I will solve with this ? :)

Reply:
Pattern names can be different across SLE versions and there is no Basis-Devel on openSUSE Tumbleweed. But this is not a big deal and can be updated anytim.

Reply:
ok